### ISSUE TITLE
Gatsby Deferred Static Generation working with Nginx

### ISSUE THREAD

**Issue by TheKnarf** [Alv-no/alv-website#613]
- Setup support for Gatsby DSG (Deferred Static Generation) inside the Docker container and make it work with Nginx.
   - Needs a start script inside the Docker container that both starts Nginx and the `gatsby serve` command. I reccomend supervisord.
   - Update ngninx config file to first go to the `gatsby serve` command and only after show a 404 page.

- Create an endpoint for invalidating / deleting the cache.
  - Hook Sanity up to call this endpoint on `publish` in Sanity
  - This bit could be split into its own task and solved in two parts.

<img width="959" alt="Screenshot 2021-12-03 at 13 40 33" src="https://user-images.githubusercontent.com/213777/144604199-bd365f54-26e0-43c5-96ca-e98ae9ad9bfe.png">

**Comment by klyngen** [comment]
I have solved the cache-invalidation using sanity-webhooks. And Steinar has implemented varnish.
